packages/web-components/src/components/content-item-horizontal/content-item-horizontal-thumbnail.ts

```diff
@@ -0,0 +1,53 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020, 2021
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement, html } from 'lit-element';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import DDSContentItem from '../content-item/content-item';
+import styles from './content-item-horizontal-thumbnail.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * A component to present content in a horizontal orientation with thumbnail.
+ *
+ * @element dds-content-item-horizontal-thumbnail
+ */
+@customElement(`${ddsPrefix}-content-item-horizontal-thumbnail`)
+class DDSContentItemHorizontalThumbnail extends DDSContentItem {
```

Comment:
I don't think this new component `<dds-content-item-horizontal-thumbnail>` is needed for the work outlined in this issue (https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/6110). It seems to me like we just need to include the [dds-mage](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/image/image.ts#L26) component within the [dds-content-item-horizontal](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/master/packages/web-components/src/components/content-item-horizontal/content-item-horizontal.ts#L25) component and defining the `image` slot within the HTML return of the shadow dom, unless there's something else that I'm missing here.

@annawen1 @photodow Can you let me know if what I'm suggesting is in alignment with what you guys are thinking as well?

https://ibm.ent.box.com/file/812734380200

Reply:
@proeung that's a good point, from the functional specs I think we could work off of `content-item-horizontal` itself and just add an attribute to render the `thumbnail` styles